src/util/storageVolume.tsx

```diff
@@ -61,12 +62,16 @@ export const getZfsVolumeFormFields = (): (keyof StorageVolumeFormValues)[] => {
 };
 
 export const getVolumeKey = (key: string): string => {
-  if (Object.keys(storageVolumeFormFieldToPayloadName).includes(key)) {
+  if (key in storageVolumeFormFieldToPayloadName) {
```

Comment:
I think this is a bit more efficient since we are only checking if the key exists in `storageVolumeFormFieldToPayloadName`